main.py

```diff
@@ -0,0 +1 @@
+print('woow')
```

Comment:
Hello! Welcome to the course!
Please do not push any extra files and work in the folder of the first lab.
Thank you!